### PR TITLE
{bio}[GCC/12.3.0] PHYLIP v3.697

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+homepage = 'https://evolution.genetics.washington.edu/phylip'
+description = "PHYLIP is a free package of programs for inferring phylogenies."
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+# -fcommon is required to avoid 'multiple definitions' linking error
+toolchainopts = {'extra_cflags': "-fcommon"}
+
+source_urls = ['https://evolution.gs.washington.edu/phylip/download/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['9a26d8b08b8afea7f708509ef41df484003101eaf4beceb5cf7851eb940510c1']
```
Comment (bedroge) [inline easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCC-12.3.0.eb:15]: I'm also getting a checksum error locally, and when inspecting the file it's basically an HTML page saying that their site has been moved.
```suggestion
source_urls = ['https://phylipweb.github.io/phylip/download/']
sources = [SOURCELOWER_TAR_GZ]
checksums = ['9a26d8b08b8afea7f708509ef41df484003101eaf4beceb5cf7851eb940510c1']
```

File: easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCC-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'MakeCp'
+
+name = 'PHYLIP'
+version = '3.697'
+
+homepage = 'https://evolution.genetics.washington.edu/phylip'
```
Comment (bedroge) [inline easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.697-GCC-12.3.0.eb:6]:
```suggestion
homepage = 'https://phylipweb.github.io/phylip'
```

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--force-download"

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**